### ISSUE TITLE
COAM\Compliance Obligation Grid - external user can see a compliance obligation ID 

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-coam#19]
#### Description:

As an external user, I can see a compliance obligation ID for each  compliance obligations 

#### Acceptance Criteria:

 - [ ] Given I am on the operation's compliance obligation page, it shows the compliance obligation ID which consists of the [emissions report ID + current year]
 - [ ] Given I am on the COAM [dashboard](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=17617-2751&t=hoovKA1meA6RBOCl-4), when I click manage compliance obligation, then it takes me to the [COAM grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=17790-2543&t=hoovKA1meA6RBOCl-4)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by NicoleGovvy** [comment]
@nanyangpro this ticket to me looks like it should be part of #17, as the compliance ID is a single data element that is visible in multiple places. Curious about your thoughts here.

**Comment by nanyangpro** [comment]
Agreed! Good call, @NicoleGovvy.
